cmake/BLTPrivateMacros.cmake

```diff
@@ -272,6 +272,46 @@ macro(blt_inherit_target_info)
 
 endmacro(blt_inherit_target_info)
 
+##------------------------------------------------------------------------------
+## blt_expand_depends( DEPENDS_ON [dep1 ...]
+##                     RESULT [variable] )
+##------------------------------------------------------------------------------
+macro(blt_expand_depends)
+    set(options)
+    set(singleValueArgs RESULT)
+    set(multiValueArgs DEPENDS_ON)
+
+    # Parse the arguments
+    cmake_parse_arguments(arg "${options}" "${singleValueArgs}"
+                        "${multiValueArgs}" ${ARGN} )
+
+    # Expand dependency list
+    set(_deps_to_process ${arg_DEPENDS_ON})
+    set(_expanded_DEPENDS_ON ${arg_DEPENDS_ON})
+    while(_deps_to_process)
+        # Copy the current set of dependencies to process
+        set(_current_deps_to_process ${_deps_to_process})
+        # and add them to the full expanded list
+        list(APPEND _expanded_DEPENDS_ON ${_deps_to_process})
```

Comment:
You are adding the passed in DEPENDS_ON twice here and on 290

Reply:
Good catch!